src/pages/index.jsx

```diff
@@ -6,18 +7,37 @@ import HomepageTestimonials from './pages-sections/homepage/homepage-testimonial
 import HomepageForm from './pages-sections/homepage/homepage-form';
 import Footer from './components-global/footer';
 
-const Home = () => (
-  <>
-    <Header />
-    <main>
-      <HomepageIntro />
-      <HomepageInfographic />
-      <HomepageCaseStudies />
-      <HomepageTestimonials />
-      <HomepageForm />
-    </main>
-    <Footer />
-  </>
+import { client } from '../lib/client';
+import { SanityProvider } from '../context/sanity-context';
+
+const Home = ({ data }) => (
+  <SanityProvider data={ data } >
+    <Fragment>
```

Comment:
@KennedyNvsf please use the shorter syntax for fragments without keys or attributes, <></> , that way you can also remove it from the import